crates/autopilot/src/domain/mod.rs

```diff
@@ -12,3 +14,9 @@ pub use {
     fee::ProtocolFee,
     quote::Quote,
 };
+
+pub struct SolvableOrders {
+    pub orders: Vec<model::order::Order>,
```

Comment:
We shouldn't refer to these orders from `domain`, not even as `boundary::Order`.

If we want to move the solvable orders type to `domain`, we should make it work solely with `domain::auction::order::Order` here, instead of `model::order::Order` which is a more involving change.

That said, I would rather remove this and make `persistence` work with `boundary::SolvableOrders` until we rewrite the whole `SolvableOrdersCache` in ddd manner and move it to `domain`.